packages/vm/src/state/stateManager.ts

```diff
@@ -57,6 +58,10 @@ export interface DefaultStateManagerOpts {
    * A {@link SecureTrie} instance
    */
   trie?: Trie
+  /**
+   * Optional custom precompiles (to exclude from access list generation)
+   */
+  customPrecompiles?: CustomPrecompile[]
```

Comment:
Hmm, I find this dependency introduction very unfortunate and rather displaced, also in regard to the StateManager extraction.

Or can we assume that `generateAccessList()` will move out of the StateManager along refactoring anyhow and that this will therefore only be a temporary addition anyhow? 🤔